substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/meta/AnalysisType.java

```diff
@@ -941,6 +941,7 @@ public int getModifiers() {
 
     @Override
     public boolean isAssignableFrom(ResolvedJavaType other) {
+        assert other != null : "other is null";
```

Comment:
This `assert` is useless, since on the next line you will get a `NullPointerException` anyways.

Reply:
True. I use the assert as an explicit marker to callers about `isAssignableFrom()` input argument.